hooks/commits.php

```diff
@@ -21,6 +22,12 @@
     . $contributing_url
     . ") for more information.";
 
+$message_sob_invalid = "<!-- PMABOT:SOB --><!-- PMABOT:SOB:INVALID -->\n"
+    . "This commit has an invalid Signed-Off-By line, please use"
+    . "your full name for legal compliance reasons.\n"
```

Comment:
You forgot to add a space here: `please useyour full name`

Reply:
Good catch !
fixed